redshift_sqlalchemy/dialect.py

```diff
@@ -643,7 +623,8 @@ def _get_all_table_and_view_info(self, connection, **kw):
             AS "diststyle",
           c.relowner AS "owner_id",
           u.usename AS "owner_name",
-          pg_get_viewdef(c.oid) AS "view_definition",
+          TRIM(TRAILING ';' FROM pg_catalog.pg_get_viewdef(c.oid, true))
```

Comment:
The output of `pg_get_viewdef` includes a semicolon at the end of the query. The SQLAlchemy `CreateTable`, etc. commands don't include a semicolon at the end of the output, so we need to strip that out somewhere for consistency.

This `TRIM(TRAILING ';'` is the solution used in psql's `\d+` command to get rid of the semicolon.